Find a functionally equivalent code:android.graphics.Picture.getWidth:COMMENT
<android.graphics.Picture: int getWidth()>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Get the width of the picture as passed to beginRecording. This
 * does not reflect (per se) the content of the picture.
 */

@@ -1 +1,3 @@
-
+{
+    return nativeGetWidth(mNativePicture);
+}

